### ISSUE TITLE
Shrink size of UINaviationBar large title text for IOS & Android

### ISSUE THREAD

**Issue by salisuabubakar** [grahammendick/navigation#728]
Hi, is there a way to change for fontsize of large title on iOS and Android ? Thanks

**Comment by grahammendick** [comment]
Hi, yea, the `titleFontSize` prop of the `NavigationBar` accepts a function. When standard is false return the font size you want for the large title
```jsx
<NavigationBar titleFontSize={standard => standard ? 14 : 24}>
```

**Comment by grahammendick** [comment]
Just remembered, it's iOS-only at the moment. Google material components didn't support custom font size on the large title. But they [added support in for it in version 1.8.0](https://github.com/material-components/material-components-android/releases/tag/1.8.0-alpha01#:~:text=CollapsingToolbarLayout) so I can add it into the Navigation router now

**Comment by grahammendick** [comment]
[Started working on it](https://github.com/grahammendick/navigation/compare/issue-728)

**Comment by grahammendick** [comment]
Just released [navigation-react-native v9.5.0 with Android support for large title font size](https://github.com/grahammendick/navigation/releases/tag/v9.5.0-NavigationReactNative). I thanked you in the release notes
